proxy/grpcproxy/watch.go

```diff
@@ -272,7 +272,8 @@ func (wps *watchProxyStream) recvLoop() error {
 			wps.delete(uv.CancelRequest.WatchId)
 			wps.lg.Debug("cancel watcher", zap.Int64("watcherId", uv.CancelRequest.WatchId))
 		default:
-			panic("not implemented")
+			// Panic or Fatalf would allow network clients to crash the serve remotely.
+			wps.lg.Error("Not support request type by gRPC proxy", zap.Stringer("request", req))
```

Comment:
NIT: I think by convention the first word does not need to be capitalized in the error message.

Reply:
DONE